tests/test_queries.py

```diff
@@ -86,6 +83,15 @@ def qmcall(c, qdict, qtype=None):
     return json
 
 
+def has_google_api_key():
+    basepath, _ = os.path.split(os.path.realpath(__file__))
+    keypath = (
+        basepath + os.sep + ".." + os.sep + "resources" + os.sep + "GoogleServerKey.txt"
+    )
```

Comment:
This should strictly speaking be done using `os.path.join()` instead of concatting `os.sep`